src/home/HomePage.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class='homepage'>
+    <tasks-list :taskList='taskList' v-on:complete='onComplete($event)' />
+    <completed-tasks-list
+      :taskList='taskList'
+      v-on:complete='onComplete($event)'
+      v-on:delete='onDelete($event)'
+    />
+  </div>
+</template>
+
+<script>
+  import CompletedTasksList from './components/CompletedTasksList.vue';
+  import TasksList from './components/TasksList.vue';
+
+  export default {
+    data() {
+      return {
+        taskList: {},
+      };
+    },
+
+    components: { TasksList, CompletedTasksList },
+    created() {
+      this.fetchTasks();
+    },
+    methods: {
+      fetchTasks() {
+        if ('taskList' in localStorage) {
+          console.log('found tasklist in localstorage');
+          this.taskList = JSON.parse(localStorage.getItem('taskList'));
+        } else {
+          const taskList = {
+            IT: [],
+            HR: [],
+            'New Hire Paperwork': [],
+            'Culture Orientation': [],
+            Other: [],
+          };
+          // {'IT':[{'title':'Do testing','assignee':'Sahil','date':'2022-08-17'}],'HR':[{'title':'Schedule a meeting','assignee':'Michael','date':'2022-08-24'}],'New Hire Paperwork':[],'Culture Orientation':[],'Other':[]}
+          localStorage.setItem('taskList', JSON.stringify(taskList));
+          this.taskList = JSON.parse(localStorage.getItem('taskList'));
+
+          console.log(JSON.parse(localStorage.getItem('taskList')));
+        }
+      },
+      onComplete(event) {
+        console.log(event);
+        const index = this.taskList[event.dept].findIndex(
+          (task) => task.id === event.id
+        );
+
+        this.taskList[event.dept][index].completed =
+          !this.taskList[event.dept][index].completed;
+        const completionDate = new Date();
```

Comment:
I think `completionDate` variable was used only once, in this case, no need to define a variable. You can directly use it.

Reply:
done